lighty-modules/lighty-gnmi/lighty-gnmi-connector/src/test/java/io/lighty/modules/gnmi/connector/tests/commons/TestUtils.java

```diff
@@ -48,4 +56,23 @@ private static String readResource(final String classPath) throws Exception {
             return CharStreams.toString(new InputStreamReader(inputStream, StandardCharsets.UTF_8));
         }
     }
+
+    private static KeyPair decodePrivateKey(final Reader reader, final String passphrase) throws IOException {
+        try (PEMParser keyReader = new PEMParser(reader)) {
```

Comment:
this method seems to be duplicate on multiple places - where is it from originally?

in addition, it dfoes not follow ODL code style  - use final, var syntax